newIDE/app/src/MarketingPlans/MarketingPlanFeatures.js

```diff
@@ -126,9 +124,11 @@ const MarketingPlanFeatures = ({
                     {selectMessageByLocale(i18n, nameByLocale)}
                   </Text>
                 </LineStackLayout>
-                <Text size="body-small" color="secondary">
-                  <Trans>{planCreditsAmount} credits</Trans>
-                </Text>
+                {planCreditsAmount && (
+                  <Text size="body-small" color="secondary">
+                    <Trans>{planCreditsAmount} credits</Trans>
+                  </Text>
+                )}
```

Comment:
those changes were to allow showing the marketing plans even if not logged in (in the performance dashboard)
I keep them for when we'll show the performance dashboard alltogether